src/mcp/antigravity/README.md

```diff
@@ -0,0 +1,113 @@
+# Firebase MCP Server
+
+[![License](https://img.shields.io/badge/License-MIT-blue.svg)](../../../LICENSE)
+
+The Firebase Model Context Protocol (MCP) Server gives AI-powered development tools the ability to work with your Firebase projects and your app's codebase. The Firebase MCP server works with any tool that can act as an MCP client, including: Antigravity, [Firebase Studio](https://firebase.google.com/studio), [Gemini CLI](https://github.com/google-gemini/gemini-cli), [Claude Code](https://www.claude.com/product/claude-code), [Cline](https://github.com/cline/cline), [Cursor](https://www.cursor.com/), VS Code Copilot, [Windsurf](https://codeium.com/windsurf), and more!
+
+## What You Can Build
+
+Use this MCP server to quickly add powerful backend features to your app:
+
+- **Add a database to your app** - Store and query your application data with Cloud Firestore or Firebase Data Connect
+- **Add user login and sign up pages** - Implement authentication flows with Firebase Authentication
+- **Host your app** - Deploy and serve your application with Firebase Hosting services
+- **Add AI logic to your app** - Build intelligent features with Firebase AI Logic
+
+Some tools use [Gemini in Firebase](https://firebase.google.com/docs/ai-assistance) to help you:
+
+- Generate Firebase Data Connect schema and operations
+- Consult Gemini about Firebase products
+
+> **Important:** Gemini in Firebase can generate output that seems plausible but is factually incorrect. It may respond with inaccurate information that doesn't represent Google's views. Validate all output from Gemini before you use it and don't use untested generated code in production. Don't enter personally-identifiable information (PII) or user data into the chat.
```

Comment:
I'd love to move this disclaimer to the bottom (maybe with an asterisk pointing users to it?)